### DISCUSSION TITLE
Time zone problem

### DISCUSSION THREAD

**Discussion by Rayjun0525** [hugo-sid/hugo-blog-awesome#40] [Q&A] [answered]
A time zone problem causes some posts not to be created normally.

My time zone is KST, which is now May 7th, 23. However, when executing 
hugo-t 'hugo-blog-awesome', 
only the post until May 6th is generated.

This issue is expected to be resolved on its own after about 9 hours, but I was wondering if I could allocate settings related to the time zone or variable values in config.toml.

**Comment by hugo-sid** [comment, marked as answer]
Hi @Rayjun0525, I think [this discussion](https://discourse.gohugo.io/t/using-date-as-a-string-in-rfc3339-format/44162/4) has the answer to your problem.

**Reply by Rayjun0525** [reply]
Beautiful!!
That was the best answer. Thank you.
**timeZone = 'America/New_York'**

**Reply by hugo-sid** [reply]
You are welcome @Rayjun0525.
